README.md

```diff
@@ -263,6 +263,46 @@ An instance of `DidEndpoint` contains the service details, i.e., service ID, a s
 
 > The validity and authenticity of the information stored on the KILT blockchain are guaranteed by the same blockchain, which verifies that all DID management operations other than creation are signed by the DID authentication key, as described in the section relative to each operation.
 
+#### Representation of Public Key Types on the DID Document
+
+Currently the use of four different types of public keys as [Verification Methods](https://w3c.github.io/did-core/#verification-methods) are supported by a full DID; light DIDs support a subset of these (see [Light DIDs](#light-dids) section for details).
+Keys type is indicated in form of an enum for both full DIDs and light DIDs.
+By reference implementations of a DID resolver compliant with the specification, keys are represented as follows on the DID document.
+
+Conforming with [DID specifications](https://w3c.github.io/did-core/#verification-methods) these Verification Methods are JSON objects containing the properties `id`, `type`, and `controller`.
```

Comment:
```suggestion
Conforming with [DID specifications](https://w3c.github.io/did-core/#verification-methods) these Verification Methods are JSON objects containing at least the properties `id`, `type`, and `controller`.
```

Reply:
It's all about context definition. So the DID spec specified what are the mandatory properties, but each DID provider could add more. Not sure what the best wording is, but you grasped by intention.